electrical/readme.MD

```diff
@@ -0,0 +1,261 @@
+
+* [SparkFun Stripping Wire](https://learn.sparkfun.com/tutorials/working-with-wire/how-to-strip-a-wire) 
+* [Splicing Wire](https://www.wikihow.com/Splice-Wire)
+* [YouTube Making Connections and Heat Shrink Tubing](https://www.youtube.com/watch?v=Y8wjv6lj5KU)
+
+**Note** Please read through this entire section and understand it fully before starting to connect devices and electronics together.
+
+## 2.1. Switch, Volt Meter, and Control Board
+There are a couple of very important things to be aware of as you are making these connections, as with any project where you are working with batteries or electrical current:
+**THE BATTERY IS LIVE AT ALL TIMES.** This means that at any point, if the two terminals of the battery come in contact they will cause an electrical short which could be very dangerous. This may trigger the protection circuit in the battery that we chose and prevent extensive damage, but we do not want to rely on that for safety. Make sure to keep the battery terminals of the +V and GND separated at all times with no possibility of contact.
+
+The battery connectors will only fit together one way. That means you have to make sure your connections are correct the first time when soldering them in: make sure to verify this before connecting anything to the battery.
+The switch we use is a Single Pole - Single Throw switch. This means that either the two terminals are connected together in one switch position or there is a physical disconnect when the switch is in the other position. Make sure to test the switch and know which are its **ON** and **OFF** positions previous to connecting it to anything. An easy way to test the switch is to use the ’Diode’ setting on your Digital Multimeter (DMM).
+It is extremely important to plug the battery in the correct direction into the volt meter as well, as plugging it in backwards (even for an instant) could damage or destroy the volt meter.
+
+Using the Tamiya connectors, connect the battery to the switch and then the volt meter according to Figure 4. The Tamiya connectors allow you to disconnect the battery from the rest of the system so you can recharge the battery.
+
+**Table 3. Necessary Parts**
+
+Item | Ref | Qty | Image 
+---- | --- | --- | ----- 
+Battery | E36 | 1 | <p align="center"> <img src="../images/components/electronics/E36.png" width="25%"> </p>
+16 AWG Wire (Black) | X2 | 1 | <p align="center"> <img src="../images/components/wiring/X2.png" width="25%"> </p>
+16 AWG Wire (Red) | X1 | 1 |  <p align="center"> <img src="../images/components/wiring/X1.png" width="25%"> </p>
+Battery Charger | E41 | 1 | <p align="center"> <img src="../images/components/electronics/E41.png" width="25%"> </p>
+Tamiya Connectors| E35 | 1 | <p align="center"> <img src="../images/components/electronics/E35.png" width="25%"> </p>
```

Comment:
Make this table 2 columns, refer to https://github.com/nasa-jpl/open-source-rover/blob/master/examples/README_example.md